tcms/testplans/views.py

```diff
@@ -50,15 +50,20 @@ def update_plan_email_settings(test_plan, form):
 # _____________________________________________________________________________
 # view functons
 
+@method_decorator(permission_required('testplans.add_testplan'), name='dispatch')
```

Comment:
This will need a test b/c there's no test which validates permissions are handled properly.